frontend/src/containers/card_container.jsx

```diff
@@ -4,6 +4,8 @@ import { connect } from 'react-redux';
 import { fetchCards, annotateCard, passCard } from '../actions/card';
 import Card from '../components/Card';
 
+console.log("Card::", Card.type);
```

Comment:
So these are in there right now because when I try to load the annotate page I get these sorts of errors:

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `Card`.
    in Card (created by CardContainer)
    in CardContainer (created by Connect(CardContainer))
    in Connect(CardContainer) (created by Smart)
    in div (created by Smart)
    in div (created by TabPane)
    in Transition (created by Fade)
    in Fade (created by TabPane)
    in TabPane (created by Tabs)
    in div (created by ForwardRef)
    in ForwardRef (created by Tabs)
    in TabContainer (created by Tabs)
    in Tabs (created by Smart)
    in Smart (created by SmartContainer)
    in SmartContainer (created by Connect(SmartContainer))
    in Connect(SmartContainer)
    in Provider
```

As far as I can tell, this is usually caused by a mismatch with the {} brackets in import/exports, or something weird in package versioning. I'm guessing it's probably the second one.

Reply:
The issue is that one of the components in the render method of `Card` is undefined. Here’s the faulty import:
```
import {
    Button,
    ButtonToolbar,
    Clearfix,
    Well,
    Tooltip,
    OverlayTrigger
} from "react-bootstrap";
```
`Well` is undefined because [it was removed](https://react-bootstrap.github.io/migrating/#well) from the library. Similarly, the removal of `Glyphicon` causes errors elsewhere.

Reply:
Thanks! I added a bunch of React-bootstrap updating fixes and now it's working.